### PR TITLE
Publish Sphinx to GitHub Pages

### PR DESCRIPTION
**Pull request by cgravill**
The docs are already live at:
https://microsoft.github.io/knossos-ksc/index.html
(it's unclear why https://microsoft.github.io/knossos-ksc/ isn't working right now, it may be when you read it. Some people claim it's a delay, others you need a few deployments)

There's some repository configuration which you do via the GitHub UI, to point to a specific branch. I used "gh-pages" which is a common default.

### CODE REVIEW

**Comment by cgravill** [comment]
blocked on #1008

**Review by awf — APPROVED** [review]
It is alive now -- yay!

The `relu3` code is missing between the paragraphs below, don't know if that's been broken for a little while:
image

**Comment by cgravill** [comment]
> It is alive now -- yay!
> 
> The `relu3` code is missing between the paragraphs below, don't know if that's been broken for a little while:
> image

I'll merge this, we should check if it's working locally.

**Pull request merged**
